### PR TITLE
[Feature] Add PowerShell language server support

### PR DESCRIPTION
**Pull request by PasiBergman**
# Description

Adds support for PowerShell language server (powershell_es) configuration

## How Has This Been Tested?

1. Installed Microsoft PowerShell: https://github.com/PowerShell/PowerShell/releases
2. Installed PowerShellEditorServices: https://github.com/PowerShell/PowerShellEditorServices/releases
3. Added configuration e.g. `lvim.lang.ps1.lsp.setup.bundle_path = "/path/to/PowerShellEditorServices"`
4. Opened a PowerShell `.ps1` file and saw powershell_es language server activated.

### CODE REVIEW

File: lua/default-config.lua [hunk 1]
```diff
@@ -1250,6 +1250,19 @@ lvim.lang = {
       },
     },
   },
+  ps1 = {
+    formatter = {},
```
Comment (abzcoding) [inline lua/default-config.lua:1254]: it should be `formatters` i think

Comment (PasiBergman) [reply lua/default-config.lua:1254]: Good point. I copy pasted the configuration above which (also) has the error. I will fix both.

**Review by abzcoding — APPROVED** [review]

**Comment by kylo252** [comment]
Nice work!
1- I guess it's not supported by LspInstall? If that's the case, maybe add the instructions somewhere, probably in `ps1.lua` as a reference at least until we figure out a better way.
2- Does this support v7 only? If so, maybe we should add a warning or a disclaimer about that.

**Comment by abzcoding** [comment]
document it [here](https://www.lunarvim.org/languages/) if you want

**Comment by PasiBergman** [comment]
@kylo252 
Good idea, I'll look into it. I think only cross-platform PowerShell (basically v7+) is supported as the LS startup requires `pwsh`.

@abzcoding 
Thx. I'll document PowerShell and C# (csharp) as they are a bit niche.

**Pull request merged**